openage/convert/main.py

```diff
@@ -170,35 +170,36 @@ def conversion_required(asset_dir, args):
     Sets options in args according to what sorts of conversion are required.
     """
     try:
-        with asset_dir['converted', ASSET_VERSION_FILENAME].open() as fileobj:
+        with asset_dir['converted',
+                       changelog.ASSET_VERSION_FILENAME].open() as fileobj:
```

Comment:
I guess it would've been preferable to `from changelog import ASSET_VERSION_FILENAME`